
The Aura Sync controller for the motherboard resides on an auxiliary SMBus.
For AMD systems, this SMBus is not initialised with the stock Linux kernel;
to get it to work, you will need to patch your kernel. The patch follows for
Linux kernel 4.20 (derived from
https://gitlab.com/CalcProgrammer1/KeyboardVisualizer/issues/85#note_121577579):

--- a/drivers/i2c/busses/i2c-piix4.c	2019-01-09 01:23:06.197945763 +1100
+++ b/drivers/i2c/busses/i2c-piix4.c	2019-01-09 01:24:58.007942622 +1100
@@ -964,6 +964,11 @@
 		retval = piix4_setup_sb800(dev, id, 1);
 	}
 
+	if (dev->vendor == PCI_VENDOR_ID_AMD &&
+	    dev->device == PCI_DEVICE_ID_AMD_KERNCZ_SMBUS) {
+		retval = piix4_setup_sb800(dev, id, 1);
+	}
+
 	if (retval > 0) {
 		/* Try to add the aux adapter if it exists,
 		 * piix4_add_adapter will clean up if this fails */
